.github/actions/verify-gha-refs/Check-Called-GHA-Refs.ps1

```diff
@@ -0,0 +1,44 @@
+param(
+    [String[]] $CalledRepoBaseIncludeList, 
+    [String[]] $PathIncludeList,
+    [String[]] $FileIncludeList,
+    [String] $ExpectedRef,
+    [String] $GitHubRepository,
+    [String] $GitHubRefName
+)
+
+if ($CalledRepoBaseIncludeList.Count -eq 0) {
+    exit 0 # Nothing to check because array is empty.
```

Comment:
Hmm. Maybe an informative log message would be good here.

Reply:
Agreed. I added a warning message; here is a screenshot from my test repository of what the summary would look like:
![image](https://github.com/Lombiq/GitHub-Actions/assets/5144137/4047a193-e7a8-42c7-bd21-5747d83834a4)

Also note, that I made an additional change to skip the `verify-gha-refs` step altogether when no GHA items (meaning actions or workflows) are part of the PR. Specifically, I added [this line](https://github.com/Lombiq/GitHub-Actions/blob/issue/OSOE-517/.github/workflows/validate-this-gha-refs-pr.yml#L39) to make the step conditional.